### ISSUE TITLE
dockerfile uses an old image

### ISSUE THREAD

**Issue by GaProgMan** [GaProgMan/PokeBlazor#15]
### Description of Issue

The dockerfile is referencing an old (no longer supported by Microsoft) dockerimage:

``` go
FROM microsoft/dotnet:2.1.0-preview2-aspnetcore-runtime-alpine
```

As such, this needs to change to be in line with the current (at the time of creating this issue) supported image fr ASP NET Core:

``` go
FROM microsoft/2.1-aspnetcore-runtime-alpine
```

**Comment by GaProgMan** [comment]
This issue has been fixed in pull request [#16](https://github.com/GaProgMan/PokeBlazor/pull/16).

**Comment by GaProgMan** [comment]
Close #16
